Build failing on Mageia 5 x86_64

Hello,
I encounter some difficulties compiling on my system, although I believe all dependencies are set up. Cmake runs ok, though it does not seem to check rust version. Here's the log:

```
$ make
[  0%] Built target gtest
CMake Error: cmake version 3.0.2
Usage: /usr/bin/cmake -E [command] [arguments ...]
Available commands: 
  chdir dir cmd [args]...   - run command in a given directory
  compare_files file1 file2 - check if file1 is same as file2
  copy file destination     - copy file to destination (either file or directory)
  copy_directory source destination   - copy directory 'source' content to directory 'destination'
  copy_if_different in-file out-file  - copy file if input has changed
  echo [string]...          - displays arguments as text
  echo_append [string]...   - displays arguments as text but no new line
  environment               - display the current environment
  make_directory dir        - create a directory
  md5sum file1 [...]        - compute md5sum of files
  remove [-f] file1 file2 ... - remove the file(s), use -f to force it
  remove_directory dir      - remove a directory and its contents
  rename oldname newname    - rename a file or directory (on one volume)
  tar [cxt][vfz][cvfj] file.tar [file/dir1 file/dir2 ...]
                            - create or extract a tar or zip archive
  sleep <number>...         - sleep for given number of seconds
  time command [args] ...   - run command and return elapsed time
  touch file                - touch a file.
  touch_nocreate file       - touch a file but do not create it.
Available on UNIX only:
  create_symlink old new    - create a symbolic link new -> old

rust/CMakeFiles/rust-build.dir/build.make:49: recipe for target 'rust/CMakeFiles/rust-build' failed
make[2]: *** [rust/CMakeFiles/rust-build] Error 1
CMakeFiles/Makefile2:649: recipe for target 'rust/CMakeFiles/rust-build.dir/all' failed
make[1]: *** [rust/CMakeFiles/rust-build.dir/all] Error 2
Makefile:137: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/rust/CMakeLists.txt
+++ b/rust/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.1)
 
 find_package(Cargo 0.9.0 REQUIRED)
 find_package(Rustc 1.8.0 REQUIRED)

